litejpeg/core/huffman/huffmancore.py

```diff
@@ -0,0 +1,479 @@
+"""
```

Comment:
Generally you don't want a space after colons.

Reply:
Done